apps/web/app/(regflow)/sign-up/ProfileScreen.tsx

```diff
@@ -0,0 +1,169 @@
+  const { profileScreen } = dict;
+  const [selectedAvatar, setSelectedAvatar] = useState("1");
+
+  return (
+    <>
+      <Form
+        onSubmit={(values) => {
+          const profileInfo = { firstName: values.firstName, lastName: values.lastName, avatar: selectedAvatar}
+          console.log(profileInfo)
+          //done(profileInfo)
+        }}>
+        {({ submit }) => (
+          <StyledForm
+            onSubmit={(e) => {
+              e.preventDefault();
+              submit();
+            }}>
+            <H3Styled>{t(profileScreen.title)}</H3Styled>
+            <SubtitleStyled>{t(profileScreen.subtitle)}</SubtitleStyled>
+            <SeparatorLine />
+            <AvatarSelector
+              avatars={[
+                {
+                  id: "1",
+                  src: "/avatars/1.svg",
+                },
+                {
+                  id: "2",
+                  src: "/avatars/2.svg",
+                },
+                {
+                  id: "3",
+                  src: "/avatars/3.svg",
+                },
+                {
+                  id: "4",
+                  src: "/avatars/4.svg",
+                },
+                {
+                  id: "5",
+                  src: "/avatars/5.svg",
+                },
+                {
+                  id: "6",
+                  src: "/avatars/6.svg",
+                },
+                {
+                  id: "7",
+                  src: "/avatars/7.svg",
+                },
+                {
+                  id: "8",
+                  src: "/avatars/8.svg",
+                },
+              ]}
+              selectedAvatar={selectedAvatar}
+              onSelect={setSelectedAvatar}
+            />
+            <Field
+              onChangeValidate={z.string().min(1, t(profileScreen.inputErrorMsg))}
+              name="firstName"
+              initialValue={""}>
+              {({ value, setValue, onBlur, errors }) => (
+                <Input
+                  name="firstName"
+                  label={t(profileScreen.firstNameInputLabel)}
+                  value={value}
+                  onChange={(e) => setValue(e.currentTarget.value)}
+                  onInputCleared={() => setValue("")}
+                  onBlur={onBlur}
+                  hasError={errors.length > 0}
+                  supportingLabel={errors.length ? errors : null}
+                />
+              )}
+            </Field>
+            <Field
+              onChangeValidate={z.string().min(1, t(profileScreen.inputErrorMsg))}
+              name="lastName"
+              initialValue={""}>
+              {({ value, setValue, onBlur, errors }) => (
+                <Input
+                  name="lastName"
+                  label={t(profileScreen.lastNameInputLabel)}
+                  value={value}
+                  onChange={(e) => setValue(e.currentTarget.value)}
+                  onBlur={onBlur}
+                  onInputCleared={() => setValue("")}
```

Comment:
```suggestion
                  onChange={(e) => setValue(e.currentTarget.value)}
                  onBlur={onBlur}
```
Clearing value also applies as a change. Is onInputCleared necessary?

Reply:
It's a input clearing option, it is provided by the input - is it neccessary? hard to tell